src/components/Dialogs/GeneralDialog/GeneralDialog.style.ts

```diff
@@ -3,13 +3,17 @@ import { css } from '@emotion/react'
 import { Button } from '@/shared/components'
 import { colors, sizes, breakpoints } from '@/shared/theme'
 
-export const StyledContainer = styled.div`
+type ContainerProps = {
+  wide?: boolean
+}
+
+export const StyledContainer = styled.div<ContainerProps>`
   position: relative;
   width: 90%;
-  max-width: ${sizes(110)};
+  max-width: ${({ wide }) => (wide ? '700px' : '440px')};
   min-height: 150px;
   margin: ${sizes(32)} auto;
-  background-color: ${colors.gray[600]};
+  background-color: ${colors.gray[700]};
```

Comment:
Plot twist: multistepper uses gray-700 but a general dailog gray-600. Must be a mistake on designers part. Let's use the `600` for everything